### ISSUE TITLE
The color contrast of the button border is less than 3:1 when setting FlatStyle to Standard

### ISSUE THREAD

**Issue by MelonWang1** [dotnet/winforms#6636]
**.NET Core Version:** 
* Main: 7.0.100-preview.2.22107.13
* Release: 7.0.100-preview.2.22107.10

**Have you experienced this same bug with .NET Framework?** 
Yes (based on [#1101503](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1101503))

**Repro steps:**

1. Create a Winforms .net core application with a button
2. Set Button.FlatStyle= FlatStyle.Standard
3. Build and run the application and use Accessibility tool to observe luminosity contrast ratio

**Actual behavior:**
Luminosity contrast ratio of the button border is less than 3:1 when Button.FlatStyle=FlatStyle.Standrad.
![31](https://user-images.githubusercontent.com/94418985/152958694-f4d7a422-ab42-44be-8b1b-16d2886350e9.png)

**Expected behavior:**

Luminosity contrast ratio of the button border is greater than 3:1.

**More info:**
1. This issue occurs in 7.0.100-preview.2.22107.13 from latest main branch and 7.0.100-preview.2.22107.10 from release branch.
2. This has been fixed and added [comment](https://github.com/dotnet/winforms/issues/6021#issuecomment-972470132) in [#6021](https://github.com/dotnet/winforms/issues/6021).

**Comment by dreddy-work** [comment]
@vladimir-krestov / @SergeySmirnov-Akvelon can you guys prioritize this regression? @MelonWang1 do you know which release of core or check-in caused this regression?

**Comment by MelonWang1** [comment]
This issue cannot repro from 7.0.100-alpha.1.21566.20 to 7.0.100-alpha.1.22069.1, and can repro from 7.0.100-preview.2.22076.3 to latest 7.0.100-preview.2.22108.9 from main branch.

**Comment by Tanya-Solyanik** [comment]
@MelonWang1 , @SergeySmirnov-Akvelon  - this is a low priority bug per accessibility export - 

Scott O'Hara
[commented Feb 14](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1101503#3005289)


```
This is a usability bug.  In this context, the text of the button should have been tested - not just the borders.  Per the WCAG understanding doc for 1.4.11:
"A button which has a distinguishing indicator such as position, text style, or context does not need a contrasting visual indicator to show that it is a button, although some users are likely to identify a button with an outline that meets contrast requirements more easily."

It would be best if the border color was updated to meet 3:1, as that would mean this would be even more distinguishable... but per what the requirement actually says / requires, this needs to be downgraded to an a11ySev4
```
